openedx/core/djangoapps/programs/models.py

```diff
@@ -132,3 +137,10 @@ def is_certification_enabled(self):
         certificates for Program completion.
         """
         return self.enabled and self.enable_certification
+
+    @property
+    def show_xseries_ad(self):
```

Comment:
TBH I'm leaning towards @schenedx's argument here.   The one fairly clean way I can see would be to have the sidebar load up a user-defined, self-contained hunk of view that is specified in the config model, and if it's empty, don't load anything.  This essentially becomes an extension to theming.  Yet it still feels under-defined to me.  I'm comfortable taking on a little debt now, pending a broader conversation about how xseries generalizes to open source users as well as for future edx.org offerings.